utils/appliance/__init__.py

```diff
@@ -1893,6 +1894,113 @@ def reset_automate_model(self):
+        self.ssh_client.run_command(
+            'cd /var/www/miq/vmdb;DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rake evm:db:reset')
+        self.ssh_client.run_rake_command('db:seed')
+        self.ssh_client.run_command('service collectd start')
+        # Work around for https://bugzilla.redhat.com/show_bug.cgi?id=1337525
+        self.ssh_client.run_command('service httpd stop')
+        self.ssh_client.run_command('rm -rf /run/httpd/*')
+        self.ssh_client.run_command('rm -rf /var/www/miq/vmdb/log/*.log*')
+        self.ssh_client.run_command('rm -rf /var/www/miq/vmdb/log/apache/*.log*')
+        self.ssh_client.run_command('service evmserverd start')
+        logger.debug('Cleaned appliance in: {}'.format(round(time() - starttime, 2)))
+
+    def set_full_refresh_threshold(self, threshold=100):
+        yaml = self.get_yaml_config()
+        yaml['ems_refresh']['full_refresh_threshold'] = threshold
+        self.set_yaml_config(yaml)
+
+    def set_cap_and_util_all_via_rails(self):
+        """Turns on Collect for All Clusters and Collect for all Datastores without using Web UI."""
+        command = (
+            'Metric::Targets.perf_capture_always = {:storage=>true, :host_and_cluster=>true};')
+        self.ssh_client.run_rails_console(command, timeout=None)
+
+    def set_cfme_server_relationship(self, vm_name, server_id=1):
+        """Set MiqServer record to the id of a VM by name, effectively setting the CFME Server
+        Relationship without using the Web UI."""
+        command = ('miq_server = MiqServer.find_by(id: {});'
+                   'miq_server.vm_id = Vm.find_by(name: \'{}\').id;'
+                   'miq_server.save'.format(server_id, vm_name))
+        self.ssh_client.run_rails_console(command, timeout=None)
+
+    def set_pglogical_replication(self, replication_type=':none'):
+        """Set pglogical replication type (:none, :remote, :global) without using the Web UI."""
+        command = ('MiqRegion.replication_type = {}'.format(replication_type))
+        self.ssh_client.run_rails_console(command, timeout=None)
+
+    def add_pglogical_replication_subscription(self, host):
+        """Add a pglogical replication subscription without using the Web UI."""
+        user = conf.credentials['ssh']['username']
+        password = conf.credentials['ssh']['password']
+        dbname = 'vmdb_production'
+        port = 5432
+        command = ('sub = PglogicalSubscription.new;'
+                   'sub.dbname = \'{}\';'
+                   'sub.host = \'{}\';'
+                   'sub.user = \'{}\';'
+                   'sub.password = \'{}\';'
+                   'sub.port = {};'
+                   'sub.save'.format(dbname, host, user, password, port))
+        self.ssh_client.run_rails_console(command, timeout=None)
+
+    def set_rubyrep_replication(self, host, port=5432, database='vmdb_production',
+                                username='root', password='v2:{I2SQ5PdmGPwN7t5goRiyaQ==}'):
```

Comment:
Use this function to encrypt the password https://github.com/ManageIQ/integration_tests/blob/320652ae129ca3e9e7c10f5cf67574715254fe12/utils/appliance/__init__.py#L512-L520
instead of doing `password='v2:{I2SQ5PdmGPwN7t5goRiyaQ==}'` :blush: